src/zitadel.rs

```diff
@@ -145,6 +166,25 @@ impl Zitadel {
 
 				if let Err(error) = status {
 					tracing::error!("Failed to update user `{}`: {:?}", new.log_name(), error);
+
+					if error.to_string().contains("invalid UpdateHumanPhoneRequest") {
+						let mut new = new.clone();
+						new.user_data.phone = None;
```

Comment:
[`assign`](https://docs.rs/assign/latest/assign/) maybe?

Reply:
I changed it to the [struct update syntax](https://doc.rust-lang.org/book/ch05-01-defining-structs.html#creating-instances-from-other-instances-with-struct-update-syntax), let me know if you insist on the `assign` macro instead.